roles/unicorn_user/templates/defaults.j2

```diff
@@ -9,3 +9,5 @@ MAIL_HOST={{ mail_host }}
 MAIL_PORT={{ mail_port }}
 SMTP_USERNAME={{ smtp_username }}
 SMTP_PASSWORD={{ smtp_password }}
+MAILS_FROM={{ mails_from | default("no-reply@{{ mail_domain }}") }}
+MAIL_BCC={{ mail_bcc | default('') }}
```

Comment:
IMO the responsibility of `ofn-install` here should only be to provide the appropriate values of these two vars to the app. Otherwise, we're duplicating logic between two repos, which will make it harder and error-prone to change.

Reply:
The values are optional, but will override the from_address of all mails from the instance. So the default is to ignore this setting, but if we leave it as an empty string I think the from_address will be set to an empty string, which is not what we want.

Reply:
I also wanted the values not to cause errors if they were missing, both in ofn-install and the main repo, because we can't know if the updates on either side will both be present at the same time.

Reply:
woudln't we have a win-win if we simply do:

```
{% if mails_from is defined %}
  MAILS_FROM={{ mails_from }}
{% endif %}
{% if mails_bcc is defined %}
  MAIL_BCC={{ mail_bcc }}
{% endif %}
```

This way the app will work in either case: providing the right value if the env vars are set or falling back to the default if they are not, while not duplicating any logic between repos. Am I right?